### ISSUE TITLE
Cannot click on a plugin to add to a aplaceholder

### ISSUE THREAD

**Issue by satyrius** [divio/django-cms#3922]
I have configured content placeholder
```python
    CMS_PLACEHOLDER_CONF = {
        'content': {
            'plugins': [
                'TextPlugin', 'PicturePlugin', 'IframePlugin',
                'LinkPlugin', 'PubDatePlugin'
            ],
            'text_only_plugins': ['LinkPlugin'],
            'name': gettext('Content'),
            'language_fallback': True,
        },
    }
```

And there is an issue with placeholder editing frontend. Look at screenshot. There are 5 plugins in the list, but the latest is hidden under scroll hint block (but there is not scroll in that control). So user cannot click on a latest plugin in a list. If I will add more plugins the scrolls appears and I can choose a plugin, but not in this case.

![2015-03-04_1832](https://cloud.githubusercontent.com/assets/278630/6486629/44890762-c29d-11e4-9ce2-3a5a0a7f9018.png)

**Comment by mkoistinen** [comment]
Hi, thanks for reporting this. Can you tell us which browser and OS?

**Comment by satyrius** [comment]
Google Chrome 40.0.2214.115, Mac OS X

**Comment by yakky** [comment]
Duplicate of #3456

This has been plaguing us for a very long time. I think this can be solved only by rethinking the structure mode UI (which is on our todo list)

**Comment by slepa** [comment]
![bug](https://cloud.githubusercontent.com/assets/807715/6739279/bcb1c8fe-ce88-11e4-9dde-59148c1b40cd.png)

I think its critical bug.
I am new to django-cms
I spend 2 hours, trying to guess how to add text to placeholder. 
Text plugin always hidden by "friendly" scroll button for me.

**Comment by mkoistinen** [comment]
Closed by #4014

**Comment by ryanbagwell** [comment]
@mkoistinen this still appears to be an issue, at least with Chrome. The threshold should be 247 instead of 245 to account for a 1px top and bottom border.
